runtime/runtime.m

```diff
@@ -1253,6 +1259,15 @@ -(void) xamarinSetFlags: (enum XamarinGCHandleFlags) flags;
 	options.EntryAssemblyPath = xamarin_entry_assembly_path;
 #endif
 
+#if defined (CORECLR_RUNTIME)
+	options.xamarin_objc_msgsend = (void *) xamarin_dyn_objc_msgSend;
+	options.xamarin_objc_msgsend_super = (void *) xamarin_dyn_objc_msgSendSuper;
+#if !defined(__aarch64__)
+	options.xamarin_objc_msgsend_stret = (void *) xamarin_dyn_objc_msgSend_stret;
+	options.xamarin_objc_msgsend_super_stret = (void *) xamarin_dyn_objc_msgSendSuper_stret;
```

Comment:
_might_ want to set them to null on arm64, not needed but cleaner

Reply:
The options struct is zero-initialized (https://github.com/xamarin/xamarin-macios/blob/4d6ab1d6c8ae555ec819efc94fa44d2c3aa0fa5d/runtime/runtime.m#L191), so setting to NULL explicitly is redundant.